gatsby-node.js

```diff
@@ -3,6 +3,23 @@ const { newCloudinary, getResourceOptions } = require('./utils');
 const REPORTER_PREFIX = `gatsby-source-cloudinary`;
 const NODE_TYPE = `CloudinaryMedia`;
 
+// Validation of configured plugin options the Gatsby way
+
+exports.pluginOptionsSchema = ({ Joi }) => {
+  return Joi.object({
+    cloudName: Joi.string().required(),
+    apiKey: Joi.string().required(),
+    apiSecret: Joi.string().required(),
+    resourceType: Joi.number().integer().positive().default(10),
+    resultsPerPage: Joi.string().default('image'),
```

Comment:
suggestion(blocking): It should be a number like max results and default should be set to the value of max result. See my comment above on how to achieve that.

Reply:
Oh I see now that you switched up `resourceType` and `resultsPerPage`.